Resources/views/_snippets/sirtrevor/method.html.twig

```diff
@@ -1,7 +1,7 @@
 {% if data.listItems is defined %}
     <ul>
         {% for item in data.listItems %}
-            <li itemprop="recipeInstructions">{{ item.content }}</li>
+            <li itemprop="recipeInstructions" itemscope itemtype="http://schema.org/HowToStep"><span itemprop="text">{{ item.content }}</span></li>
```

Comment:
Hi @maxblack72 
What would happen if I have the JSON-ld bundle on my page with this patch? 
What would happen if my Json-ld says something and this patch spit something completely different?

Reply:
Also, the recipe instructions are now part of the CMS. 
Look at the bottom of this drink page: https://cms.idlcloud.com/www.plymouthgin.com/en-EN/drink/556/show

Reply:
1. I don't know but you might need to pull the new tag to your project maybe? Maybe @hermzz  can help. After a couple of hours of research I found the correct way to represent these info for google so I wanted to fix that once for all. This is the default template anyway, we generally override on every project in the app/Resources/SirTrevorTwig/views/_snippets/sirtrevor/ folder. 
2. this snippet won't affect category, calories and all the other fields you mentioned. this will affect the only the list of the methods.